src/foam/nanos/ndiff/NDiffDAO.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * @license
+ * Copyright 2021 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+    package: 'foam.nanos.ndiff',
+    name: 'NDiffDAO',
+    extends: 'foam.dao.ProxyDAO',
+
+    doumentation: `
+    Decorator for an existing DAO that logs puts to the ndiffDAO service
+    if it's running. 
+    `,
+    javaImports: [
+        'foam.nanos.logger.Logger',
```

Comment:
Can you set your editor to 2 spaces. Thanks.

Reply:
all ndiff sourcefiles converted to two space - further issues, if any, to be corrected down the line